### PR TITLE
ISPN-15240 Improve cross-site message handling

### PR DESCRIPTION
**Pull request by pruivo**
https://issues.redhat.com/browse/ISPN-15240

### CODE REVIEW

**Comment by pruivo** [comment]
Should fix in `XSiteAutoStateTransferTest` random failures.

File: core/src/main/java/org/infinispan/util/logging/Log.java [hunk 1]
```diff
@@ -2375,4 +2376,7 @@ CacheConfigurationException storeConfiguredHasBothReadAndWriteOnly(String storeC
    @LogMessage(level = WARN)
    @Message(value = "Failed to register metrics with id %s. Reason: %s", id = 970)
    void metricRegistrationFailed(String id, String reason);
+
+   @Message(value = "Simple caches do not support queries. Use standard caches for querying.", id = 971)
```
Comment (ryanemerson) [inline core/src/main/java/org/infinispan/util/logging/Log.java:2380]: This message doesn't seem appropriate for the state we're trying to describe :slightly_smiling_face:

Comment (pruivo) [reply core/src/main/java/org/infinispan/util/logging/Log.java]: Let me fire the AI bot that wrote it :rofl:

**Comment by pruivo** [comment]
Let's put this on hold until QE tests it.

**Comment by ryanemerson** [comment]
Thanks @pruivo

**Pull request merged**